RecoTauTag/RecoTau/src/PFRecoTauClusterVariables.cc

```diff
@@ -235,6 +235,111 @@ namespace reco {
           }
         }
 
+        if (mvaOpt == kDBnewDMwLTwGJPhase2) {
+          mvaInput[0] = tau.pt();
+          mvaInput[1] = std::abs(tau.eta());
+          mvaInput[2] = chargedIsoPtSum;               //tauID("chargedIsoPtSum");
+          mvaInput[3] = neutralIsoPtSum;               //tauID("neutralIsoPtSum");
+          mvaInput[4] = puCorrPtSum;                   //tauID("puCorrPtSum");
+          mvaInput[5] = photonPtSumOutsideSignalCone;  //tauID("photonPtSumOutsideSignalCone");
+          mvaInput[7] = tauDecayMode;                  //tau.decayMode();
```

Comment:
I suppose this was meant to be 6.
I guess this is the origin of random results in jenkins tests

Reply:
Another example of the need for #29818...